src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -99,54 +101,83 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
 
     const hotInstanceColHeaders: string[] = (newStudentsHOTInstance.getColHeader() as string[]);
 
-    const studentsEnrollRequest: StudentsEnrollRequest = {
-      studentEnrollRequests: [],
-    };
+    const allStudentChunks: StudentEnrollRequest[][] = [];
+    let currentStudentChunk: StudentEnrollRequest[] = [];
+    const numberOfStudentsPerRequest: number = 50;
```

Comment:
slightly magic to a first time viewer. Can put some comment to explain to the future viewers what 50 entails?